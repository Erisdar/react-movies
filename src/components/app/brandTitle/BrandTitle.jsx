import React from 'react';
import './brandTitle.css';

export default ({ className }) => <p className={className}>netflixroulette</p>