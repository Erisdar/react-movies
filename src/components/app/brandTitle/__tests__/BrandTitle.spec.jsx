import { shallow } from 'enzyme';
import React from 'react';
import BrandTitle from '../BrandTitle';

describe('BrandTitle component', () => {
    it('should be render correctly', () => {
        const component = shallow(<BrandTitle />);
        expect(component).toMatchSnapshot();
    });
});