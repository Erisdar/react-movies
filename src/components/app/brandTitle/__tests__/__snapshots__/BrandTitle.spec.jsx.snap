// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrandTitle component should be render correctly 1`] = `
<p>
  netflixroulette
</p>
`;
